packages/release/mask-local-mnt.service

```diff
@@ -0,0 +1,21 @@
+[Unit]
+Description=Mask Local Mnt Directory (/local/mnt)
+DefaultDependencies=no
+RequiresMountsFor=/local /mnt
+BindsTo=mnt.mount
+ConditionPathIsMountPoint=!/local/mnt
+Conflicts=umount.target
+Before=umount.target
+RefuseManualStart=true
+RefuseManualStop=true
+
+[Service]
+Type=oneshot
+
+# Mask `/local/mnt` to avoid confusion, since it will have most of the contents
+# of `/mnt` but not any of the mounts.
+ExecStart=/usr/bin/mount --bind --options nosuid,nodev,noexec,private /srv /local/mnt
```

Comment:
TIL `/srv`, somewhat unexpected way of using it.

Reply:
It's definitely odd. I figured relying on an empty `/srv` as an implementation detail was a less grievous sin against [FHS](https://refspecs.linuxfoundation.org/FHS_3.0/fhs/index.html) than inventing a top-level `/empty` directory to serve the same purpose.

I'm open to other suggestions if this is too weird.